_posts/2022-02-28-war-in-ukraine.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "The War in Ukraine"
+author: [michal.kosmulski]
+tags: [tech, "soft skills", helping]
+---
+The attack on Ukraine has shocked the whole world, but in Poland it is perceived in an especially poignant way.
```

Comment:
**Very minor:** I think that "_entire_" fits better here ([example explanation](https://english.stackexchange.com/questions/9615/whole-vs-entire#:~:text=%E2%80%9CWhole%E2%80%9D%20comes%20from%20%E2%80%9Cunhurt,are%20collections%20of%20distinct%20objects.)) 🙂 
```suggestion
The attack on Ukraine has shocked the entire world, but in Poland it is perceived in an especially poignant way.
```